frame/staking/src/mock.rs

```diff
@@ -200,7 +200,7 @@ impl Trait for Test {
 	type Time = pallet_timestamp::Module<Self>;
 	type Event = ();
 	type SessionsPerEra = SessionsPerEra;
-	type BondingDurationInEra = ();
+	type BondingDurationInEra = BondingDurationInEra;
```

Comment:
Here is why we got the `cannot slash at` or `BondingDurationInEra` issue, the [initial][0] `BondingDurationInEra` just not correct for our tests, and the sessions get removed(fn `[new_era][1]`) even in the first 3 era.

[0]: https://github.com/paritytech/substrate/blob/013c1ee167354a08283fb69915fda56a62fee943/frame/staking/src/mock.rs#L213

[1]: https://github.com/darwinia-network/darwinia/blob/6155b46e00588d98845eaa2357b3562d05a588f5/frame/staking/src/lib.rs#L1955

Reply:
I forget to add modify this line.